komodo/release_transpiler.py

```diff
@@ -249,5 +264,17 @@ def main():
     args.func(args)
 
 
+def valid_file(path: str) -> str:
+    if os.path.isfile(path):
+        return path
+    raise TypeError(path)
+
+
+def dir_path(should_be_valid_path: str) -> str:
+    if os.path.isdir(should_be_valid_path):
+        return should_be_valid_path
+    raise TypeError(should_be_valid_path)
```

Comment:
ditto

Reply:
I will change this one to NotADirectoryError